ion/symboltoken_test.go

```diff
@@ -0,0 +1,236 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package ion
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func newString(value string) *string {
+	return &value
+}
+
+var boolEqualsTestData = []struct {
+	text1   *string
+	sid1    int64
+	source1 *ImportSource
+	text2   *string
+	sid2    int64
+	source2 *ImportSource
+}{
+	{nil, 123, nil,
+		nil, 123, nil},
+	{nil, 123, newSource("table", 1),
+		nil, 123, newSource("table", 1)},
+	{newString("text1"), 123, nil,
+		newString("text1"), 123, nil},
+	{newString("text2"), 123, newSource("table", 1),
+		newString("text2"), 123, newSource("table", 1)},
+}
```

Comment:
Indentation looks off to me (maybe its the PR view in the browser?)
Even though LocalSID is not taken into account for equality, the test should have test cases were the value differs

Reply:
Updated.